src/colmap/controllers/automatic_reconstruction.cc

```diff
@@ -49,9 +50,9 @@ AutomaticReconstructionController::AutomaticReconstructionController(
     : options_(options),
       reconstruction_manager_(std::move(reconstruction_manager)),
       active_thread_(nullptr) {
-  CHECK(ExistsDir(options_.workspace_path));
-  CHECK(ExistsDir(options_.image_path));
-  CHECK_NOTNULL(reconstruction_manager_);
+  THROW_CHECK(ExistsDir(options_.workspace_path));
+  THROW_CHECK(ExistsDir(options_.image_path));
+  THROW_CHECK(reconstruction_manager_);
```

Comment:
THROW_CHECK_NOTNULL

Reply:
The current code inconsistently uses `CHECK` and `CHECK_NOTNULL` for pointers. Should we switch all to `CHECK_NOTNULL` for consistency?

Reply:
OK, I was just looking at the first file and saw the discrepancy.... :-) Unless the THROW_CHECK_NOTNULL would be much more informative, it doesn't really matter imo.